clang/include/clang/ASTMatchers/ASTMatchFinder.h

```diff
@@ -135,10 +135,15 @@ class MatchFinder {
       llvm::StringMap<llvm::TimeRecord> &Records;
     };
 
+    MatchFinderOptions() {}
```

Comment:
`MatchFinderOptions() = default;` (why add it in the first place? Having no ctor should be enough)